custom_components/ha_washdata/frontend.py

```
@@ -326,6 +327,22 @@ async def async_register_panel(hass: HomeAssistant) -> bool:
                     _LOGGER.debug("Panel translations path registration failed: %s", exc)
                     _register_static_path(hass, PANEL_TRANSLATIONS_URL, str(trans_src))
+
+            # Serve brand/icon.png so the panel header can display the real icon.
+            icon_src = Path(__file__).parent / "brand" / "icon.png"
+            if await hass.async_add_executor_job(icon_src.exists):
+                try:
+                    from homeassistant.components.http import StaticPathConfig  # pylint: disable=import-outside-toplevel
+
+                    if hasattr(hass.http, "async_register_static_paths"):
+                        await hass.http.async_register_static_paths(
+                            [StaticPathConfig(BRAND_ICON_URL, str(icon_src), True)]
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)